OasisDEX/oasis-borrow
# Add History Coming Soon to AAVE manage view

[SC-10898]

# [[Frontend] New tab on aave manage page](https://app.shortcut.com/oazo-apps/story/10898/frontend-new-tab-on-aave-manage-page)
  
## Changes 👷‍♀️
- Add disabled VaultHistory Control for coming soon state
- Add Aave history support by network config
  
## How to test 🧪
- Go to a position such as ethereum/aave/v2/178#history
- Switch networks and observe behaviour (all currently set to false anyway for now) but undefined networks will show no tab
